Tools/Scripts/webkitpy/port/gtk.py

```diff
@@ -81,7 +82,7 @@ def setup_environ_for_server(self, server_name=None):
                                                                     ignore_errors=True).strip()
                 dri_libgl_path = os.path.join(llvmpipe_libgl_path, "dri")
             else:  # in flatpak
-                llvmpipe_libgl_path = "/usr/lib/x86_64-linux-gnu/"
+                llvmpipe_libgl_path = "/usr/lib/{}-linux-gnu/".format(platform.machine())
```

Comment:
You can use also `os.uname().machine` and avoid the import of the platform module.